### DISCUSSION TITLE
export collention failed

### DISCUSSION THREAD

**Discussion by yu929** [outline/outline#5144] [Self Hosting]
When I tried to export the document set, an error occurred: "certificate has expired". The specific error message in the backend is as follows:
{"error":"certificate has expired","stack":"Error: certificate has expired\n    at TLSSocket.onConnectSecure (node:_tls_wrap:1530:34)\n    at TLSSocket.emit (node:events:526:28)\n    at TLSSocket.emit (node:domain:475:12)\n    at TLSSocket._finishInit (node:_tls_wrap:944:8)\n    at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:725:12)","level":"error","message":"Error processing task in ExportMarkdownZipTask"}

**Comment by routerino** [comment]
.... has your certificate for outline or S3 expired?

**Reply by yu929** [reply]
I was able to resolve this issue on my own. As I deployed the application on an internal network, the certificate was self-signed. I set NODE_TLS_REJECT_UNAUTHORIZED to 0, and the problem was resolved.
